.github/workflows/reusable_upload_web.yml

```diff
@@ -151,10 +151,12 @@ jobs:
 
             - [ ] I have tested the web viewer
 
-            | Result | Commit  | Link  |
-            | ------ | ------- | ----- |
+            | Result | Commit  | Link |
+            | ------ | ------- | ---- |
             | ✅ | ${{ steps.get-sha.outputs.sha }} | https://rerun.io/viewer/pr/${{ github.event.pull_request.number }} |
 
+            If you're updating examples, you might want to look at the following manifests: [`main`](https://rerun.io/viewer/pr/${{ github.event.pull_request.number }}?manifest_url=https://app.rerun.io/version/main/examples_manifest.json), [`nightly`](https://rerun.io/viewer/pr/${{ github.event.pull_request.number }}?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
```

Comment:
Personally I'd prefer the comment to be as short as possible, because we'll be seeing it in almost every PR. I put stuff in a table, because more structure makes it easier to find what you're looking for at a glance

Reply:
Yeah, you're right! I've accepted your suggestion!